editor/src/components/canvas/canvas-strategies/strategies/set-flex-gap-strategy.tsx

```diff
@@ -172,30 +172,33 @@ export const setFlexGapStrategy: CanvasStrategyFactory = (
       }
 
       if (shouldTearOffGap) {
-        return strategyApplicationResult([
-          deleteProperties('always', selectedElement, [StyleGapProp]),
-        ])
+        return strategyApplicationResult(
+          [deleteProperties('always', selectedElement, [StyleGapProp])],
+          'rerender-all-elements',
```

Comment:
possibly one to fix here too

Reply:
Added comment about follow up in 38ec355d5.